client-java/controller/src/main/java/org/evomaster/client/java/controller/problem/rpc/RPCExceptionHandler.java

```diff
@@ -125,11 +125,11 @@ private static boolean handleThrift(Object e, EndpointSchema endpointSchema, RPC
             handled = handleTException(e, dto);
 
             if (!handled){
-                SimpleLogger.error("Fail to extract exception type info for an exception "+ e.getClass().getName());
+                SimpleLogger.uniqueWarn("Fail to extract exception type info for an exception "+ e.getClass().getName());
             }
 
         } catch (ClassNotFoundException ex) {
-            SimpleLogger.error("ERROR: in handling Thrift exception with error msg:"+ex.getMessage());
+            SimpleLogger.uniqueWarn("ERROR: in handling Thrift exception with error msg:"+ex.getMessage());
```

Comment:
are those `errors` (faults in EM) or `warnings` (other kinds of issues that are not critical)?  it is weird to see a `ERROR: ...` txt in a `warn` message